### ISSUE TITLE
Password not copied to clipboard after locking the screen

### ISSUE THREAD

**Issue by dpoetzsch** [dpoetzsch/keyman#7]
This happens for me now too on Gnome 3.32 (Fedora 30) after I've locked my screen and unlock it again. For some reason it doesn't put passwords in the clipboard anymore after that.

When I lock the keyring and then unlock it again it works again.

But then when I lock the screen again, again it doesn't work anymore.

I see this error in `journalctl -xe`:

```
JS ERROR: Exception in callback for signal: activate: Gio.IOErrorEnum: GDBus.Error:org.freedesktop.S>
_proxyInvoker@resource:///org/gnome/gjs/modules/overrides/Gio.js:138:41
_makeProxyMethod/<@resource:///org/gnome/gjs/modules/overrides/Gio.js:164:16
KeyringConnection.prototype._getSecret@/home/rpkamp/.local/share/gnome-shell/extensions/keyman@dpoetz>
KeyringConnection.prototype.getSecretFromPath/<@/home/rpkamp/.local/share/gnome-shell/extensions/keym>
KeyringConnection.prototype.unlockObject@/home/rpkamp/.local/share/gnome-shell/extensions/keyman@dpoe>
KeyringConnection.prototype.getSecretFromPath@/home/rpkamp/.local/share/gnome-shell/extensions/keyman>
_copySecret@/home/rpkamp/.local/share/gnome-shell/extensions/keyman@dpoetzsch.github.com/keyman.js:15>
_createSecretMenuItem/<@/home/rpkamp/.local/share/gnome-shell/extensions/keyman@dpoetzsch.github.com/>
_emit@resource:///org/gnome/gjs/modules/signals.js:142:27
activate@resource:///org/gnome/shell/ui/popupMenu.js:167:9
_onButtonReleaseEvent@resource:///org/gnome/shell/ui/popupMenu.js:118:9
```

_Originally posted by @rpkamp in https://github.com/dpoetzsch/keyman/issues/2#issuecomment-498674686_

**Comment by dpoetzsch** [comment]
This bug was introduced when doing the refactoring that fixed #5 (commit 34bf3dff13e5ed9a321010b58bb9ed8919e9d67c).

**Comment by rpkamp** [comment]
I can confirm this is fixed in `master` currently :+1:
